aerich/utils.py

```diff
@@ -101,3 +104,43 @@ def import_py_file(file: Union[str, Path]) -> ModuleType:
     module = importlib.util.module_from_spec(spec)  # type:ignore[arg-type]
     spec.loader.exec_module(module)  # type:ignore[union-attr]
     return module
+
+
+def diff_fields(
+    old_fields: list[dict], new_fields: list[dict], ident_key="through"
```

Comment:
I think `ident_key` can be just key similarly to `key` in `sorted`. `ident_key` isn't a common name.